dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/event/RumEventSerializer.kt

```diff
@@ -8,61 +8,44 @@ package com.datadog.android.rum.internal.domain.event
 
 import com.datadog.android.core.internal.domain.Serializer
 import com.datadog.android.rum.RumAttributes
+import com.google.gson.Gson
 import com.google.gson.JsonArray
 import com.google.gson.JsonNull
 import com.google.gson.JsonObject
 import com.google.gson.JsonPrimitive
-import java.io.PrintWriter
-import java.io.StringWriter
-import java.text.SimpleDateFormat
 import java.util.Date
-import java.util.Locale
-import java.util.TimeZone
 
 internal class RumEventSerializer : Serializer<RumEvent> {
 
-    private val simpleDateFormat = SimpleDateFormat(ISO_8601, Locale.US).apply {
-        timeZone = TimeZone.getTimeZone("UTC")
-    }
+    private val gson: Gson = Gson()
 
     // region Serializer
 
     override fun serialize(model: RumEvent): String {
-        val root = JsonObject()
-
-        // Event Context
-        val context = model.context
-        root.addProperty(RumAttributes.APPLICATION_ID, context.applicationId)
-        root.addProperty(RumAttributes.SESSION_ID, context.sessionId)
-        root.addProperty(RumAttributes.VIEW_ID, context.viewId)
-
-        // Timestamp
-        val formattedDate = simpleDateFormat.format(Date(model.timestamp))
-        root.addProperty(RumAttributes.DATE, formattedDate)
+        val json = gson.toJsonTree(model.event).asJsonObject
 
         // User Info
-        addUserInfo(model, root)
+        addUserInfo(model, json)
 
         // Network Info
-        addNetworkInfo(model, root)
-
-        // Data
-        addEventData(model.eventData, root)
+        addNetworkInfo(model, json)
 
         // custom attributes
-        addCustomAttributes(model, root)
+        addCustomAttributes(model, json)
 
-        return root.toString()
+        println("SERIALIZED JSON\n$json\n")
```

Comment:
We should not print this in production.

Reply:
Oups yes this was a remnant of something else